### PR TITLE
BESS AP-403+AP-404

### PR DESCRIPTION
**Pull request by MicheleBeargroup**
Addresses non simple-path related fixes, formatting and file name changes with BESS PR.

### CODE REVIEW

File: src/Payment/Gateway/Http/Client/SettlementClient.php [hunk 1]
```diff
@@ -20,8 +20,7 @@
 
 /**
  * Class SettlementClient
- *
- * @package Amazon\Payment\Gateway\Http\Client
+ * Amazon Pay capture client
```
Comment (prayagr) [inline src/Payment/Gateway/Http/Client/SettlementClient.php]: @MicheleBeargroup , should this be Amazon Pay capture client? I see a similar comment in src/Payment/Gateway/Http/Client/CaptureClient.php above.

Comment (MicheleBeargroup) [reply src/Payment/Gateway/Http/Client/SettlementClient.php]: When I did the sweep to change Amazon Payment to Amazon Pay, I also included comments so I was trying to be consistent.

File: src/Payment/Domain/Validator/AmazonAuthorization.php [hunk 2]
```diff
@@ -21,7 +21,7 @@
 
 /**
  * Class AmazonAuthorization
- * @package Amazon\Payment\Domain\Validator
+ * validates Amazon Pay status during cron updates.
```
Comment (lemked) [inline src/Payment/Domain/Validator/AmazonAuthorization.php]: A bit inconsistent as when compared to other docs (should use capital letter and no end point)

File: src/Payment/Gateway/Http/Client/CaptureClient.php [hunk 3]
```diff
@@ -19,8 +19,7 @@
 
 /**
  * Class Client
- *
- * @package Amazon\Payment\Gateway\Http\Client
+ * Amazon Pay gateway capture client
  */
 class CaptureClient extends AbstractClient
```
Comment (lemked) [inline src/Payment/Gateway/Http/Client/CaptureClient.php]: Shouldn't we add a try..catch using a logger for the CaptureClient's process method?

File: src/Payment/Gateway/Http/Client/AuthorizeClient.php [hunk 4]
```diff
@@ -19,8 +19,7 @@
 
 /**
  * Class Client
- *
- * @package Amazon\Payment\Gateway\Http\Client
+ * Amazon Pay authorization gateway client
  */
 class AuthorizeClient extends AbstractClient
```
Comment (lemked) [inline src/Payment/Gateway/Http/Client/AuthorizeClient.php]: Shouldn't we add a try..catch using a logger for the AuthorizeClient's process method?

**Pull request merged**